wcag20/sources/quickref.xslt

```diff
@@ -652,7 +652,7 @@ script type="text/javascript" src="annotate.js" xmlns="http://www.w3.org/1999/xh
     </xsl:for-each>
   </xsl:template>
   <xsl:template match="p[@role='i'] | p[@role='v']">
-    <p  >
+    <p>
       <xsl:if test="@id">
         <xsl:attribute name="id"><xsl:value-of select="@id"/></xsl:attribute>
```

Comment:
i'm assuming/hoping that in this case the extra space wasn't actually needed to avoid the xsl operation just cramming in that attribute without space, i.e. `<pname="...">`